### PR TITLE
Added script for GeeksForGeeks submissions

### PR DESCRIPTION
**Pull request by BRO3886**
This is the first time I'm writing code for/contributing to an extension 🙈

# What works
- getting content from page
- sync submissions
- GeeksForGeeks does not return any proper stats so only sending const text messages

# Known issues
- [x] ~too many requests to the API, need to fix the setTimeout~
- [x] ~`extension context invalidated` sometimes (trying to see why that might be happening)~
- [x] ~unable to find the id of button on DOM sometimes~

@QasimWani any suggestions welcome!

### CODE REVIEW

**Comment by mdhishaamakhtar** [comment]
@QasimWani would love to see this merged

File: scripts/leetcode.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const languages = {
   Java: '.java',
   'C#': '.cs',
   JavaScript: '.js',
+  Javascript: '.js',
```
Comment (andythsu) [inline scripts/leetcode.js:10]: Isn't this line same as the one above?

Comment (BRO3886) [reply scripts/leetcode.js:10]: Nope, geeksforgeeks treats it as Java**s**cript while leetcode treats it as Java**S**cript

Comment (andythsu) [reply scripts/leetcode.js:10]: Ahhh I see. What a subtle difference

**Review by QasimWani — APPROVED** [review]
looks good. merged!

**Pull request merged**